### DISCUSSION TITLE
Minecraft出新版本了

### DISCUSSION THREAD

**Discussion by zhoujy1209** [Meloong-Git/PCL#1226] [提问] [answered]
23w06a啥时候更新啊，籽岷已经拍了版本介绍视频了

**Comment by WForst-Breeze** [comment, marked as answer]
请在 **设置-启动器-下载-版本列表获取** 选择 **尽量使用官方源**，即可第一时间获取到版本更新~
- 请注意，镜像源更新速度较慢，所以经常会出现看不到新版本的情况。如果您的网络环境较好，可以使用上述解决方案；如果网络环境不佳，推荐使用镜像源，但可能需要等待一段时间后才能下载最新版本。

**Reply by WForst-Breeze** [reply]
官方源确实下载速度慢，但镜像源更新速度慢（）鱼和熊掌不可兼得嘛 (/▽＼)
